doc/whats_new.rst

```diff
@@ -85,6 +88,12 @@ API
   :meth:`~hnn_core.extracellular.ExtracellularArray.plot_lfp`, by
   `Steven Brandt`_ and `Ryan Thorpe`_ in :gh:`517`.
 
+- Recorded voltages/currents from the soma, as well all sections, are enabled by
+  setting either `record_vsoma` and `record_isoma`, or `record_vsec` and `record_isec`
```

Comment:
As I also suggested below, why do we need `record_vsoma` and `record_isoma`?

Reply:
Thought process here is 
1) Consistency with old API (although I'm renaming the attributes of `net.cell_response` here so perhaps it's a moot point)
2) Have to imagine there's going to be a memory issue with larger networks or some computational burden when storing all this data. Could be nice to not store _all_ currents/voltages if you only care about those in the soma